p2plending/lending/factories.py

```diff
@@ -55,9 +55,23 @@ class Meta:
     item = factory.RelatedFactory(ItemFactory,"title")
     author = factory.Sequence(lambda n: fuzz_name(n))
     publish_year = factory.fuzzy.FuzzyInteger(1700,now().year)
-
-
-# Todo maybe make this a factoryboy custom fuzzy attr
+class LoanFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Loan
+    borrower = factory.SubFactory(ProfileFactory)
+    item = factory.SubFactory(ItemFactory)
+    start_date = now()
+    due_date = now()+timedelta(days=LOAN_PERIOD)
+    status = random.choice([x[0] for x in LOAN_STATUS])
```

Comment:
Can you please add an empty line in between classes. In general, PEP8 is the "official" python styleguide, and it is good to generally follow it:  https://www.python.org/dev/peps/pep-0008/#blank-lines

(Although I am sure I don't always get it right, one alternative is to add a "linter" to your dev environment)

Reply:
I just fixed it. Is there anything else?